libbeat/processors/conditionals.go

```diff
@@ -108,3 +109,79 @@ func addCondition(
 
 	return NewConditionRule(condConfig, p)
 }
+
+type ifThenElseConfig struct {
+	Cond conditions.Config `config:"if"   validate:"required"`
+	Then *common.Config    `config:"then" validate:"required"`
+	Else *common.Config    `config:"else"`
+}
+
+// IfThenElseProcessor executes one set of processors (then) if the condition is
+// true and another set of processors (else) if the condition is false.
+type IfThenElseProcessor struct {
+	cond conditions.Condition
+	then *Processors
+	els  *Processors
+}
```

Comment:
Oh yeah `else` is indeed a reserved word. We can keep `els` or we could use `consequent' and `alternative` that define the content of the `Then` and the `Else`

Reply:
If it's ok with you I'm going to keep it as is. I like the fields being "close" to what's used in the config.

Reply:
Ok for me, when I saw _els_, I was like or Andrew made a typo and after I was like he didn't it just a reserved word :)